terraform/modules/happy-tfe-okta-service-account/.validate/main.tf

```diff
@@ -0,0 +1,18 @@
+module "test_validate" {
+  source = "../../happy-tfe-okta-service-account"
+
+  tags = {
+    env       = "test"
+    managedBy = "teste"
+    owner     = "test"
+    project   = "test"
+    service   = "test"
+  }
+  providers = {
+    aws.czi-si = aws.czi-si
+  }
+}
+
+provider "aws" {
```

Comment:
This is how it is expecting the module be called, so you just have to pass in the alias so that validation on this module works properly. Think of this like a unit test. Please, no more skipping testing.